spec/marker-layer-spec.coffee

```diff
@@ -176,24 +176,34 @@ describe "MarkerLayer", ->
       expect(layer2.findMarkers(containsPoint: [0, 4])).toEqual [layer2Marker]
 
   describe "::onDidUpdate", ->
-    it "notifies observers asynchronously when markers are created, updated, or destroyed", (done) ->
+    it "notifies observers at the end of the outermost transaction when markers are created, updated, or destroyed", ->
+      [marker1, marker2] = []
+
       updateCount = 0
       layer1.onDidUpdate ->
         updateCount++
         if updateCount is 1
```

Comment:
Super minor comment, but now that the event fires synchronously, do you think we could restructure this test to be simpler? Like:

```js
let updateCount = 0
layer1.onDidUpdate(() => updateCount++)

// first update
// second update
// etc.
```